runhouse/resources/hardware/cluster.py

```diff
@@ -185,7 +183,17 @@ def config_for_rns(self):
             ],
         )
         if self.is_up():
-            config["ssh_creds"] = self.ssh_creds
+            creds = (
```

Comment:
just for clarity, can we call this `creds_string` or something? was confused when reading